--- org/pSQL.py	2010-01-16 11:47:44.000000000 +0100
+++ pat/pSQL.py	2010-01-16 11:48:32.000000000 +0100
@@ -18,10 +18,7 @@
 from string import split, join, strip, find
 from types import StringType, IntType, LongType, DictionaryType
 from types import ListType, FloatType, TupleType
-try:
-  from mx.DateTime import DateTimeType
-except:
-  from DateTime import DateTimeType
+from datetime import datetime
 
 import sys
 
@@ -988,7 +985,7 @@
 def _qstr(t):
   try:
     if type(t) == StringType: s = escape(t)
-    elif type(t) == DateTimeType: s = t.strftime("%Y-%m-%d %H:%M:%S")
+    elif type(t) == datetime: s = t.strftime("%Y-%m-%d %H:%M:%S")
     else: s = escape(str(t))
   except:
     s = ''
