 macaddress/fields.py | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/macaddress/fields.py b/macaddress/fields.py
index d924850..d9af6fe 100644
--- a/macaddress/fields.py
+++ b/macaddress/fields.py
@@ -9,10 +9,9 @@ from . import default_dialect, format_mac, mac_linux
 
 import warnings
                  
-class MACAddressField(models.Field):
+class MACAddressField(models.Field, metaclass=models.SubfieldBase):
     description = "A MAC address validated by netaddr.EUI"
     empty_strings_allowed = False
-    __metaclass__ = models.SubfieldBase
     dialect = None
     
     def __init__(self, integer=True, *args, **kwargs): # Custom __init__ to accept new "integer" boolean argument to support specification of string or integer storage. Defaults to True (for now).
