diff -Nu -Nu -r NagiosGrapher-1.6.1-rc5-0.5.orig/config.layout NagiosGrapher-1.6.1-rc5-0.5/config.layout
--- NagiosGrapher-1.6.1-rc5-0.5.orig/config.layout	2007-08-15 12:23:16.000000000 +0200
+++ NagiosGrapher-1.6.1-rc5-0.5/config.layout	2008-04-19 20:23:29.000000000 +0200
@@ -189,3 +189,27 @@
 	ng_logos:			/nagios/images/logos
 	init_script_dir:		/etc/init.d
 </Layout>
+
+<Layout gentoo>
+	prefix:				/usr/
+	ng_config:                      /etc/nagios
+	nagios_config:			${ng_config}/nagios.cfg
+	nagios_config_cgi:		${ng_config}/cgi.cfg
+	nagios_contribution:		${prefix}/contrib
+	nagios_folder_cgi:		${prefix}/sbin
+	nagios_images:			${prefix}/nagios/share/images/
+	nagios_images_logos:		${prefix}/nagios/share/images/logos
+	ng_cgi:				${prefix}/sbin
+	ng_config_sub:			${prefix}/nagios/ngraph.d
+	ng_daemon:			/var/run/nagios-grapher
+	ng_config_cgi:			${ng_config}/cgi.cfg
+	ng_interface_pipe:		/var/nagios/rw/ngraph.pipe
+	ng_logfile:			/var/nagios/ngraph.log
+	ng_logos:			${prefix}/nagios/share/images/logos
+	ng_rrd:				/var/nagios/ngraph.rrd.d
+	ng_rrd_font:			/usr/share/rrdtool/fonts/DejaVuSansMono-Roman.ttf
+	ng_srvext_dir:			${ng_config}/serviceextinfo.d
+	ng_srvext_file:			${ng_config}/serviceextinfo.cfg
+	perl_inc:			/etc/perl
+	init_script_dir:		/etc/init.d
+</Layout>
