diff -Nu -Nu -r NagiosGrapher-1.6.1-rc5-0.5.orig/Makefile.in NagiosGrapher-1.6.1-rc5-0.5/Makefile.in
--- NagiosGrapher-1.6.1-rc5-0.5.orig/Makefile.in	2007-08-23 13:49:20.000000000 +0200
+++ NagiosGrapher-1.6.1-rc5-0.5/Makefile.in	2008-04-19 20:42:39.000000000 +0200
@@ -214,9 +214,6 @@
 	gcc contrib/udpecho/udpecho.c -o \
 		contrib/udpecho/udpecho
 
-	chown -R $(NAGIOS_PROC_USER) $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-	chgrp -R $(NAGIOS_PROC_GROUP) $(DESTDIR)/$(NAGIOS_CONTRIBUTION)
-
 perl-install:
 	mkdir -p $(PERL_INC)/NagiosGrapher/Hooks
 
@@ -228,8 +225,8 @@
 	chgrp -R $(NAGIOS_PROC_GROUP) $(DESTDIR)/$(PERL_INC)
 
 init-script:
-	cp nagios_grapher $(DESTDIR)/$(NG_INIT_SCRIPT_DIR)
-	chmod 0755 $(DESTDIR)/$(NG_INIT_SCRIPT_DIR)/nagios_grapher
+	cp nagios-grapher $(DESTDIR)/$(NG_INIT_SCRIPT_DIR)
+	chmod 0755 $(DESTDIR)/$(NG_INIT_SCRIPT_DIR)/nagios-grapher
 
 image-install:
 	mkdir -p $(DESTDIR)/$(NAGIOS_IMAGES)
