--- NagiosGrapher-1.6.1-rc5-0.5.orig/configure  2008-04-19 18:52:16.000000000 +0200
+++ NagiosGrapher-1.6.1-rc5-0.5/configure       2008-04-19 18:52:03.000000000 +0200
@@ -2031,7 +2031,7 @@
                s/^#.*$//m;
                s/^\s+//gim;
                s/\s+$/\n/gim;
-               s/\+$/\/nagios_grapher/gim;
+               s/\+$/\/nagios-grapher/gim;
                # m4 will not let us just use $srcdir/config.layout, we need $1
                s/^\s*(nagios_config|ng_config_sub|nagios_config_cgi|nagios_images|nagios_images_logos|nagios_folder_cgi|perl_inc|nagios_contribution|ng_config|ng_daemon|ng_srvext_dir|ng_rrd)\s*:\s*(.*)$/$1=$2/gim;
                s/^\s*(.*?)\s*:\s*(.*)$/\(test "x\$$1" = "xNONE" || test "x\$$1" = "x") && $1=$2/gim;
@@ -2390,12 +2390,12 @@
               { echo "$as_me:$LINENO: result: $ac_res" >&5
 echo "${ECHO_T}$ac_res" >&6; }
 if test `eval echo '${'$as_ac_File'}'` = yes; then
-  cp $withval ./nagios_grapher.in
+  cp $withval ./nagios-grapher.in
 else
 
-               initFileDirAndPrefix=init_scripts/nagios_grapher
+               initFileDirAndPrefix=init_scripts/nagios-grapher
                defaultSuffix=default
-               targetFile=./nagios_grapher.in
+               targetFile=./nagios-grapher.in
                initFile=$initFileDirAndPrefix.default
                as_ac_File=`echo "ac_cv_file_$initFile" | $as_tr_sh`
 { echo "$as_me:$LINENO: checking for $initFile" >&5
@@ -2429,9 +2429,9 @@
 
 else
 
-               initFileDirAndPrefix=init_scripts/nagios_grapher
+               initFileDirAndPrefix=init_scripts/nagios-grapher
                defaultSuffix=default
-               targetFile=./nagios_grapher.in
+               targetFile=./nagios-grapher.in
                initFile=$initFileDirAndPrefix.$DISTRO_NAME
                as_ac_File=`echo "ac_cv_file_$initFile" | $as_tr_sh`
 { echo "$as_me:$LINENO: checking for $initFile" >&5
@@ -2612,7 +2612,7 @@
 #AC_ARG_WITH(ng-dir-daemon,
 #      AC_HELP_STRING([--with-ng-dir-daemon=DIR], [the folder for the daemon data]),
 #      NG_DAEMON=$withval,
-#      NG_DAEMON=/usr/local/nagios/var/nagios_grapher
+#      NG_DAEMON=/usr/local/nagios/var/nagios-grapher
 #)
 #AC_SUBST(NG_DAEMON)
 
@@ -2734,7 +2734,7 @@
 
 
 # GENERATE THE FILES ...
-ac_config_files="$ac_config_files Makefile bin/collect2.pl bin/fifo_write.pl lib/NagiosGrapher.pm lib/NagiosGrapher/HTML.pm lib/NagiosGrapher/Hooks/Generic.pm lib/NagiosGrapher/Hooks/SrvExtWriteHostextInfo.pm sbin/graphs.cgi sbin/rrd2-graph.cgi sbin/rrd2-system.cgi nagios_grapher cfg/ngraph.ncfg"
+ac_config_files="$ac_config_files Makefile bin/collect2.pl bin/fifo_write.pl lib/NagiosGrapher.pm lib/NagiosGrapher/HTML.pm lib/NagiosGrapher/Hooks/Generic.pm lib/NagiosGrapher/Hooks/SrvExtWriteHostextInfo.pm sbin/graphs.cgi sbin/rrd2-graph.cgi sbin/rrd2-system.cgi nagios-grapher cfg/ngraph.ncfg"
 
 
 cat >confcache <<\_ACEOF
@@ -3315,7 +3315,7 @@
     "sbin/graphs.cgi") CONFIG_FILES="$CONFIG_FILES sbin/graphs.cgi" ;;
     "sbin/rrd2-graph.cgi") CONFIG_FILES="$CONFIG_FILES sbin/rrd2-graph.cgi" ;;
     "sbin/rrd2-system.cgi") CONFIG_FILES="$CONFIG_FILES sbin/rrd2-system.cgi" ;;
-    "nagios_grapher") CONFIG_FILES="$CONFIG_FILES nagios_grapher" ;;
+    "nagios-grapher") CONFIG_FILES="$CONFIG_FILES nagios-grapher" ;;
     "cfg/ngraph.ncfg") CONFIG_FILES="$CONFIG_FILES cfg/ngraph.ncfg" ;;
 
   *) { { echo "$as_me:$LINENO: error: invalid argument: $ac_config_target" >&5
