diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/clock/clock.c xview-3.2p1.4/clients/clock/clock.c
--- xview-3.2p1.4.orig/clients/clock/clock.c	2005-01-28 13:27:53.000000000 -0800
+++ xview-3.2p1.4/clients/clock/clock.c	2006-03-15 19:57:00.000000000 -0800
@@ -54,6 +54,11 @@
 #define		DIGITAL_ON		0
 
 extern double rint();
+static int min();
+static int armwidth();
+static int seconds_on();
+static int date_on();
+
 
 static int initializing;
 static int centerX, centerY;
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/gif.c xview-3.2p1.4/clients/olvwm-4.1/gif.c
--- xview-3.2p1.4.orig/clients/olvwm-4.1/gif.c	2005-01-28 13:27:50.000000000 -0800
+++ xview-3.2p1.4/clients/olvwm-4.1/gif.c	2006-03-15 19:57:00.000000000 -0800
@@ -57,6 +57,9 @@
 } Gif89 = { -1, -1, -1, 0 };
 
 extern XImage* ReadImage();
+static int ReadColorMap();
+static int DoExtension();
+static int GetDataBlock();
 
 XImage *ReadGIF(dpy, fd, pNcolors, pColors)
 Display *dpy;
@@ -456,7 +459,7 @@
 	return code;
 }
 
-static XImage*
+extern XImage*
 ReadImage(dpy, fd, len, height, cmap, interlace, ignore)
 Display *dpy;
 FILE    *fd;
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/olvwmrc.y xview-3.2p1.4/clients/olvwm-4.1/olvwmrc.y
--- xview-3.2p1.4.orig/clients/olvwm-4.1/olvwmrc.y	2005-01-28 13:27:50.000000000 -0800
+++ xview-3.2p1.4/clients/olvwm-4.1/olvwmrc.y	2006-03-15 19:57:00.000000000 -0800
@@ -37,7 +37,7 @@
 char	*LookupToken();
 char	*FindOlvwmRC();
 void	resetKeys();
-char	*strexpand();
+static char	*strexpand();
 
 typedef struct progscreen {
     char        *target;
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/screen.c xview-3.2p1.4/clients/olvwm-4.1/screen.c
--- xview-3.2p1.4.orig/clients/olvwm-4.1/screen.c	2005-01-28 13:27:50.000000000 -0800
+++ xview-3.2p1.4/clients/olvwm-4.1/screen.c	2006-03-15 19:57:00.000000000 -0800
@@ -153,6 +153,14 @@
 static XrmQuark virtualPixmapColorCQ;
 static XrmQuark virtualPixmapColorIQ;
 
+static updateScreenWorkspaceColor();
+static updateScreenWindowColor();
+static updateScreenForegroundColor();
+static updateScreenBackgroundColor();
+static updateScreenBorderColor();
+static updateScreenGlyphFont();
+static updateScreenInputFocusColor();
+
 /*-------------------------------------------------------------------------
  *	Local Functions
  *-------------------------------------------------------------------------*/
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/st.c xview-3.2p1.4/clients/olvwm-4.1/st.c
--- xview-3.2p1.4.orig/clients/olvwm-4.1/st.c
+++ xview-3.2p1.4/clients/olvwm-4.1/st.c
@@ -39,6 +39,8 @@
 
 #define do_hash(key, table) (*table->hash)(key, table->num_bins)
 
+static rehash(st_table *table);
+
 /*#define do_hash(key, table)\
     ((table->hash == ST_PTRHASH) ? (((int) (key) >> 2) % table->num_bins) :\
 	(table->hash == ST_NUMHASH) ? ((int) (key) % table->num_bins) :\
#diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/virtual.c xview-3.2p1.4/clients/olvwm-4.1/virtual.c
#--- xview-3.2p1.4.orig/clients/olvwm-4.1/virtual.c	2005-01-28 13:33:12.000000000 -0800
#+++ xview-3.2p1.4/clients/olvwm-4.1/virtual.c	2006-03-15 19:57:00.000000000 -0800
#@@ -30,6 +30,8 @@
# #include "vdm.icon"
# #include "vdm.mask"
# 
#+static regerr();
#+
# #if (defined(__linux__)||defined(__CYGWIN__)) && defined(__GLIBC__) && 0
# /* GNU libc doesn't use INIT, so we have to define sp ourselves. We have to
#  * initialize it as well before we call compile(). Maybe this is a bug in
#@@ -68,6 +70,8 @@
# 		*MakeUpRightButton(),*MakeRightButton(),*MakeDownRightButton();
# 
# extern char	*ExpandPath();
#+static rexMatch();
#+static rexInit();
# 
# /*
#  * Semantic action associated with each of the buttons above.  These must
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/winframe.c xview-3.2p1.4/clients/olvwm-4.1/winframe.c
--- xview-3.2p1.4.orig/clients/olvwm-4.1/winframe.c	2005-01-28 13:27:50.000000000 -0800
+++ xview-3.2p1.4/clients/olvwm-4.1/winframe.c	2006-03-15 19:57:00.000000000 -0800
@@ -84,6 +84,7 @@
 static void setIMStatusText();
 #endif
 static void updateResizePositions();
+static headerHeight();
 
 /***************************************************************************
 * sizing and decoration positioning functions
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olvwm-4.1/winicon.c xview-3.2p1.4/clients/olvwm-4.1/winicon.c
--- xview-3.2p1.4.orig/clients/olvwm-4.1/winicon.c	2005-01-28 13:27:50.000000000 -0800
+++ xview-3.2p1.4/clients/olvwm-4.1/winicon.c	2006-03-15 19:57:00.000000000 -0800
@@ -52,6 +52,10 @@
 * forward-declared functions
 ***************************************************************************/
 
+static int heightTopIcon();
+static int heightBottomIcon();
+static int widthBothIcon();
+
 /***************************************************************************
 * private event functions
 ***************************************************************************/
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/cmdstream.c xview-3.2p1.4/clients/olwm/cmdstream.c
--- xview-3.2p1.4.orig/clients/olwm/cmdstream.c	2005-01-28 13:28:03.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/cmdstream.c	2006-03-15 19:57:00.000000000 -0800
@@ -51,10 +51,10 @@
  *      Local Forward Declarations
  * ---------------------------------------------------------------------*/
 
-Command		*MatchCommand();
-CmdAttr		*MatchAttr();
-int		EncodeAttrValue();
-int		DecodeAttrValue();
+static Command		*MatchCommand();
+static CmdAttr		*MatchAttr();
+static int		EncodeAttrValue();
+static int		DecodeAttrValue();
 
 /* ----------------------------------------------------------------------
  *      SetCmdStream
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/menu.c xview-3.2p1.4/clients/olwm/menu.c
--- xview-3.2p1.4.orig/clients/olwm/menu.c	2005-01-28 13:28:02.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/menu.c	2006-03-15 19:57:00.000000000 -0800
@@ -105,11 +105,11 @@
 static void drawButton();
 static void drawRevButton();
 static Bool isClick();
+static Bool isEnabled();
 
 int MenuTrack();
 void DrawLocCursor();
 
-
 /*
  *****************************************************************************
  * Code for dealing with storing menu defaults.
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/screen.c xview-3.2p1.4/clients/olwm/screen.c
--- xview-3.2p1.4.orig/clients/olwm/screen.c	2005-01-28 13:28:02.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/screen.c	2006-03-15 19:57:00.000000000 -0800
@@ -111,6 +111,12 @@
 static XrmQuark stippledRubberBandsCQ;
 static XrmQuark stippledRubberBandsIQ;
 
+static updateScreenWorkspaceColor();
+static updateScreenWindowColor();
+static updateScreenForegroundColor();
+static updateScreenBackgroundColor();
+static updateScreenBorderColor();
+static updateScreenGlyphFont();
 
 /*-------------------------------------------------------------------------
  *	Local Functions
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/slave.c xview-3.2p1.4/clients/olwm/slave.c
--- xview-3.2p1.4.orig/clients/olwm/slave.c	2005-01-28 13:33:44.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/slave.c	2006-03-15 19:57:00.000000000 -0800
@@ -44,7 +44,7 @@
  *	Local Forward Declarations
  * ---------------------------------------------------------------------*/
 
-void	SlaveFailure();
+static void	SlaveFailure();
 
 
 /* ----------------------------------------------------------------------
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/st.c xview-3.2p1.4/clients/olwm/st.c
--- xview-3.2p1.4.orig/clients/olwm/st.c	2005-01-28 13:28:02.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/st.c	2006-03-15 19:57:00.000000000 -0800
@@ -34,6 +34,8 @@
 	(table->hash == ST_NUMHASH) ? ((int) (key) % table->num_bins) :\
 	(*table->hash)((key), table->num_bins))
 
+static rehash();
+
 st_table *st_init_table_with_params(compare, hash, size, density, grow_factor,
 				    reorder_flag)
 int (*compare)();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/winframe.c xview-3.2p1.4/clients/olwm/winframe.c
--- xview-3.2p1.4.orig/clients/olwm/winframe.c	2005-01-28 13:28:02.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/winframe.c	2006-03-15 19:57:00.000000000 -0800
@@ -72,6 +72,7 @@
 static void setIMStatusText();
 #endif
 static void updateResizePositions();
+static int headerHeight();
 
 /***************************************************************************
 * sizing and decoration positioning functions
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwm/winicon.c xview-3.2p1.4/clients/olwm/winicon.c
--- xview-3.2p1.4.orig/clients/olwm/winicon.c	2005-01-28 13:28:02.000000000 -0800
+++ xview-3.2p1.4/clients/olwm/winicon.c	2006-03-15 19:57:00.000000000 -0800
@@ -31,6 +31,9 @@
 
 extern 	Bool	PropGetWMName();
 extern 	Bool	PropGetWMIconName();
+static int heightTopIcon();
+static int heightBottomIcon();
+static int widthBothIcon();
 
 /***************************************************************************
 * private data
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwmslave/cmdstream.c xview-3.2p1.4/clients/olwmslave/cmdstream.c
--- xview-3.2p1.4.orig/clients/olwmslave/cmdstream.c	2005-01-28 13:27:52.000000000 -0800
+++ xview-3.2p1.4/clients/olwmslave/cmdstream.c	2006-03-15 19:57:00.000000000 -0800
@@ -50,10 +50,10 @@
  *      Local Forward Declarations
  * ---------------------------------------------------------------------*/
 
-Command		*MatchCommand();
-CmdAttr		*MatchAttr();
-int		EncodeAttrValue();
-int		DecodeAttrValue();
+static Command		*MatchCommand();
+static CmdAttr		*MatchAttr();
+static int		EncodeAttrValue();
+static int		DecodeAttrValue();
 
 /* ----------------------------------------------------------------------
  *      SetCmdStream
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwmslave/helprecv.c xview-3.2p1.4/clients/olwmslave/helprecv.c
--- xview-3.2p1.4.orig/clients/olwmslave/helprecv.c	2005-01-28 13:27:52.000000000 -0800
+++ xview-3.2p1.4/clients/olwmslave/helprecv.c	2006-03-15 19:57:00.000000000 -0800
@@ -15,7 +15,7 @@
 #include "helpcmd.h"
 
 extern	int	ShowHelpWindow();
-	void	ReceiveHelpCmd();
+static	void	ReceiveHelpCmd();
 
 /* ----------------------------------------------------------------------
  *	RegisterHelpWindow
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwmslave/helpwin.c xview-3.2p1.4/clients/olwmslave/helpwin.c
--- xview-3.2p1.4.orig/clients/olwmslave/helpwin.c	2005-01-28 13:27:52.000000000 -0800
+++ xview-3.2p1.4/clients/olwmslave/helpwin.c	2006-03-15 19:57:00.000000000 -0800
@@ -128,14 +128,15 @@
  *	Local Forward Functions
  * ---------------------------------------------------------------------*/
 int		ShowHelpWindow();
-void		InitHelpWindow();
-Bool		CreateHelpWindow();
-Notify_value	DestroyHelpWindow();
-void		ResetHelpWindow();
-Bool		TextHelpWindow();
-Bool		ImageHelpWindow();
-void		MoreHelp();
-void		ErrorNotice();
+static void		InitHelpWindow();
+static Bool		CreateHelpWindow();
+static Notify_value	DestroyHelpWindow();
+static void		ResetHelpWindow();
+static Bool		TextHelpWindow();
+static Bool		ImageHelpWindow();
+static void		MoreHelp();
+static void		ErrorNotice();
+static int		ConstrainMousePos();
 
 /* ----------------------------------------------------------------------
  *	ShowHelpWindow
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwmslave/olwmslave.c xview-3.2p1.4/clients/olwmslave/olwmslave.c
--- xview-3.2p1.4.orig/clients/olwmslave/olwmslave.c	2005-01-28 13:27:52.000000000 -0800
+++ xview-3.2p1.4/clients/olwmslave/olwmslave.c	2006-03-15 19:57:00.000000000 -0800
@@ -44,9 +44,9 @@
 /* ----------------------------------------------------------------------
  *	Forward Definitions
  * ----------------------------------------------------------------------*/
-void		InitScreenInfo();
-void		ParseScreenArgs();
-Notify_value	InputReader();
+static void		InitScreenInfo();
+static void		ParseScreenArgs();
+static Notify_value	InputReader();
 
 /* ----------------------------------------------------------------------
  * 	main
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwmslave/propsrecv.c xview-3.2p1.4/clients/olwmslave/propsrecv.c
--- xview-3.2p1.4.orig/clients/olwmslave/propsrecv.c	2005-01-28 13:27:52.000000000 -0800
+++ xview-3.2p1.4/clients/olwmslave/propsrecv.c	2006-03-15 19:57:00.000000000 -0800
@@ -15,7 +15,7 @@
 #include "propscmd.h"
 
 extern	int	ShowWindowProps();
-	void	ReceivePropsCmd();
+static	void	ReceivePropsCmd();
 
 /* ----------------------------------------------------------------------
  *	RegisterPropsWindow
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/olwmslave/propswin.c xview-3.2p1.4/clients/olwmslave/propswin.c
--- xview-3.2p1.4.orig/clients/olwmslave/propswin.c	2005-01-28 13:27:52.000000000 -0800
+++ xview-3.2p1.4/clients/olwmslave/propswin.c	2006-03-15 19:57:00.000000000 -0800
@@ -38,9 +38,9 @@
  *      Local Forward Declarations
  * ----------------------------------------------------------------------*/
 int		ShowWindowProps();
-void		InitWindowProps();
-Bool		CreateWindowProps();
-Notify_value	DestroyWindowProps();
+static void		InitWindowProps();
+static Bool		CreateWindowProps();
+static Notify_value	DestroyWindowProps();
 void		ApplyProps();
 void		ResetProps();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/clients/textedit/textedit.c xview-3.2p1.4/clients/textedit/textedit.c
--- xview-3.2p1.4.orig/clients/textedit/textedit.c	2005-01-28 13:33:31.000000000 -0800
+++ xview-3.2p1.4/clients/textedit/textedit.c	2006-03-15 19:57:00.000000000 -0800
@@ -51,6 +51,7 @@
 void	frame_cmdline_help();
 static int		off();
 static Notify_value	mysigproc();
+static die();
 /*
 extern Notify_value	textedit_event_proc();
 */
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/attr/attr_copy.c xview-3.2p1.4/lib/libxview/attr/attr_copy.c
--- xview-3.2p1.4.orig/lib/libxview/attr/attr_copy.c	2005-01-28 13:29:04.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/attr/attr_copy.c	2006-03-15 19:57:00.000000000 -0800
@@ -12,6 +12,11 @@
 
 #include <xview/attr.h>
 
+static int copy_1_attr(),
+	    copy_counted_list(),
+	    copy_singleton(),
+	    copy_null_list();
+
 /*
  * attr_copy:	copy an attribute list, returning the size in bytes
  */
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/dnd/dnd.c xview-3.2p1.4/lib/libxview/dnd/dnd.c
--- xview-3.2p1.4.orig/lib/libxview/dnd/dnd.c	2005-01-28 13:29:36.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/dnd/dnd.c	2006-03-15 19:57:00.000000000 -0800
@@ -22,7 +22,11 @@
 static int  SendDndEvent(),
 	    Verification(),
 	    ConstructSiteList(),
-	    FindDropSite();
+	    FindDropSite(),
+	    SendTrigger(),
+	    SendOldDndEvent(),
+	    WaitForAck(),
+	    IsV2App();
 
 /* DND_HACK begin */
 /* The code highlighted by the words DND_HACK is here to support dropping
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/dnd/dnd_dsdm.c xview-3.2p1.4/lib/libxview/dnd/dnd_dsdm.c
--- xview-3.2p1.4.orig/lib/libxview/dnd/dnd_dsdm.c	2005-01-28 13:29:36.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/dnd/dnd_dsdm.c	2006-03-15 19:57:00.000000000 -0800
@@ -26,7 +26,7 @@
 			(dnd->siteRects[i].screen_number == dnd->screenNumber)
 
 Xv_private int DndSendPreviewEvent();
-
+static void ReplyProc();
 
 Xv_private int
 DndContactDSDM(dnd)
@@ -40,7 +40,6 @@
     if (!dnd->sel) {
         Xv_object  	owner,
     			server;
-    	static void	ReplyProc();
 
     	owner = (Xv_object)xv_get(DND_PUBLIC(dnd), XV_OWNER);
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/file_chooser/file_list.c xview-3.2p1.4/lib/libxview/file_chooser/file_list.c
--- xview-3.2p1.4.orig/lib/libxview/file_chooser/file_list.c	2005-01-28 13:35:14.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/file_chooser/file_list.c	2006-03-15 19:57:00.000000000 -0800
@@ -41,6 +41,7 @@
 static int		flist_list_notify();
 static int		validate_new_directory();
 static void		flist_new_dir();
+static int		flist_match_regex();
 
 #if defined(__STDC__) || defined(__cplusplus) || defined(c_plusplus)
 static void	flist_error(File_list_private *private, char *format, ...);
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/filter/filter.c xview-3.2p1.4/lib/libxview/filter/filter.c
--- xview-3.2p1.4.orig/lib/libxview/filter/filter.c	2005-01-28 13:36:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/filter/filter.c	2006-03-15 19:57:00.000000000 -0800
@@ -39,11 +39,11 @@
 
 char           *stream_fgets();
 
-enum CharClass
+static enum CharClass
                 breakProc();
-struct CharAction
+static struct CharAction
                 digits();
-
+static          any_shell_meta_filter();
 
 struct filter_rec **
 xv_parse_filter_table(in, filename)
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/font/font.c xview-3.2p1.4/lib/libxview/font/font.c
--- xview-3.2p1.4.orig/lib/libxview/font/font.c	2005-01-28 13:29:37.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/font/font.c	2006-03-15 19:57:00.000000000 -0800
@@ -133,6 +133,7 @@
 #endif
 
 Xv_private Xv_Font xv_font_with_name();
+static int      font_construct_name();
 static void     font_default_font();
 static char    *font_default_font_from_scale();
 static char    *font_determine_font_name();
@@ -144,8 +145,10 @@
 static int	font_delim_count();
 static XID font_try_misc_name();
 static Font_locale_info *find_font_locale_info();
+static int  font_setup_defaults();
 static int	font_setup_known_families();
 static int	font_setup_known_styles();
+static int  font_init_create_attrs();
 static void	font_init_known_families();
 static void	font_init_known_styles();
 static void	font_init_sizes();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/frame/fmcmd_set.c xview-3.2p1.4/lib/libxview/frame/fmcmd_set.c
--- xview-3.2p1.4.orig/lib/libxview/frame/fmcmd_set.c	2005-01-28 13:29:26.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/frame/fmcmd_set.c	2006-03-15 19:57:00.000000000 -0800
@@ -18,6 +18,8 @@
 #include <xview/panel.h>
 #include <xview/server.h>
 
+static int update_default_pin_state();
+
 Pkg_private     Xv_opaque
 frame_cmd_set_avlist(frame_public, avlist)
     Frame           frame_public;
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/io_stream/input.c xview-3.2p1.4/lib/libxview/io_stream/input.c
--- xview-3.2p1.4.orig/lib/libxview/io_stream/input.c	2005-01-28 13:29:09.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/io_stream/input.c	2006-03-15 19:57:00.000000000 -0800
@@ -22,7 +22,7 @@
 		XV_MSG("input stream not of type input"), 0); \
 	ops = in->ops.input_ops
 
-char	*stream_getstring();
+static char	*stream_getstring();
 
 /* GENERIC INPUT FUNCTIONS */
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/misc/gettext.c xview-3.2p1.4/lib/libxview/misc/gettext.c
--- xview-3.2p1.4.orig/lib/libxview/misc/gettext.c	2005-01-28 13:37:55.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/misc/gettext.c	2006-03-15 19:57:00.000000000 -0800
@@ -11,7 +11,7 @@
 
 char *strdup();
 char * dgettext(), *bindtextdomain();
-char *_gettext();
+static char *_gettext();
 char *fgets(), *getenv();
 caddr_t mmap();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/misc/quark.c xview-3.2p1.4/lib/libxview/misc/quark.c
--- xview-3.2p1.4.orig/lib/libxview/misc/quark.c	2005-01-28 13:29:31.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/misc/quark.c	2006-03-15 19:57:00.000000000 -0800
@@ -16,7 +16,7 @@
 #include  <xview/xview.h>
 #include  <xview/pkg.h>
 
-Xv_opaque resource_type_conv();
+static Xv_opaque resource_type_conv();
 
 /* 
  *    Utilities to deal with quark lists and such.
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/notice/notice_pt.c xview-3.2p1.4/lib/libxview/notice/notice_pt.c
--- xview-3.2p1.4.orig/lib/libxview/notice/notice_pt.c	2005-01-28 13:36:20.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/notice/notice_pt.c	2006-03-15 19:57:00.000000000 -0800
@@ -63,6 +63,7 @@
 static void     notice_copy_event();
 static Xv_window	notice_get_focus_win();
 static int	notice_show_focus_win();
+static int notice_quadrant();
 
 /*
  * --------------------------- STATICS ------------------------------
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/openwin/ow_view.c xview-3.2p1.4/lib/libxview/openwin/ow_view.c
--- xview-3.2p1.4.orig/lib/libxview/openwin/ow_view.c	2005-01-28 13:36:16.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/openwin/ow_view.c	2006-03-15 19:57:00.000000000 -0800
@@ -52,6 +52,7 @@
 static          openwin_test_for_sb();
 static          openwin_link_view();
 static int      openwin_unlink_view();
+static int      openwin_check_view();
 static          openwin_locate_right_viewers();
 static          openwin_locate_left_viewers();
 static          openwin_locate_bottom_viewers();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/item_set.c xview-3.2p1.4/lib/libxview/panel/item_set.c
--- xview-3.2p1.4.orig/lib/libxview/panel/item_set.c	2005-01-28 13:28:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/item_set.c	2006-03-15 19:57:00.000000000 -0800
@@ -25,6 +25,7 @@
 Xv_private Graphics_info *xv_init_olgx();
 
 static void item_adjust_label_size();
+static fix_positions();
 
 extern Notify_value panel_base_event_handler();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/panel.c xview-3.2p1.4/lib/libxview/panel/panel.c
--- xview-3.2p1.4.orig/lib/libxview/panel/panel.c	2005-01-28 13:35:07.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/panel.c	2006-03-15 19:57:00.000000000 -0800
@@ -27,6 +27,7 @@
 Pkg_private int panel_destroy();
 
 static int      panel_layout();
+static          panel_unregister_view();
 
 #if !defined(__linux__) && !defined(__CYGWIN__)
 Xv_private_data Defaults_pairs xv_kbd_cmds_value_pairs[4];
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/p_choice.c xview-3.2p1.4/lib/libxview/panel/p_choice.c
--- xview-3.2p1.4.orig/lib/libxview/panel/p_choice.c	2005-01-28 13:28:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/p_choice.c	2006-03-15 19:57:00.000000000 -0800
@@ -113,9 +113,13 @@
 static void     choice_menu_done_proc();
 static void	choice_update_focus_win();
 static void	compute_nrows_ncols();
+static int  find_choice();
 static void     layout_choices();
+static int      modify_choice();
+static int      move_specified();
 static void     paint_choice();
 static void     preview_choice();
+static int      re_alloc_choices();
 static void     update_display();
 static void     update_item_rect();
 static void     update_value_rect();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/p_gauge.c xview-3.2p1.4/lib/libxview/panel/p_gauge.c
--- xview-3.2p1.4.orig/lib/libxview/panel/p_gauge.c	2005-01-28 13:35:01.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/p_gauge.c	2006-03-15 19:57:00.000000000 -0800
@@ -45,6 +45,7 @@
 static void	gauge_layout();
 
 /* Local functions */
+static int      etoi_gauge();
 static void     paint_gauge();
 static void     update_rects();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/p_get.c xview-3.2p1.4/lib/libxview/panel/p_get.c
--- xview-3.2p1.4.orig/lib/libxview/panel/p_get.c	2005-01-28 13:28:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/p_get.c	2006-03-15 19:57:00.000000000 -0800
@@ -15,6 +15,7 @@
 #include <xview/font.h>
 #include <xview/scrollbar.h>
 
+static          panel_shrink_margin();
 static int      shrink_to_fit();
 
 #define MAX_NEGATIVE_SHRINK 2000
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/p_set.c xview-3.2p1.4/lib/libxview/panel/p_set.c
--- xview-3.2p1.4.orig/lib/libxview/panel/p_set.c	2005-01-28 13:28:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/p_set.c	2006-03-15 19:57:00.000000000 -0800
@@ -23,7 +23,8 @@
 Xv_private char     *xv_font_regular_cmdline();
 
 static void panel_set_fonts();
-
+static int column_from_absolute_x();
+static int row_from_absolute_y();
 
 Pkg_private     Xv_opaque
 panel_set_avlist(panel_public, avlist)
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/panel/p_slider.c xview-3.2p1.4/lib/libxview/panel/p_slider.c
--- xview-3.2p1.4.orig/lib/libxview/panel/p_slider.c	2005-01-28 13:34:58.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/panel/p_slider.c	2006-03-15 19:57:00.000000000 -0800
@@ -51,7 +51,9 @@
 /* Local functions */
 static void	adjust_slider();
 static void	check_endbox_entered();
+static int  etoi_slider();
 static Panel_setting get_value();
+static int      itoe();
 static void     paint_slider();
 static void     update_rects();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/rect/rect_util.c xview-3.2p1.4/lib/libxview/rect/rect_util.c
--- xview-3.2p1.4.orig/lib/libxview/rect/rect_util.c	2005-01-28 13:28:34.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/rect/rect_util.c	2006-03-15 19:57:00.000000000 -0800
@@ -16,6 +16,8 @@
 
 #include <xview/rect.h>
 
+static int rect_nearest_edge();
+
 /*
  * Compute the distance from rect to (x, y). If (x, y) is in rect, zero is
  * returned. If x_used or y_used are non-zero, the projection point is
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/sel/sel_agent.c xview-3.2p1.4/lib/libxview/sel/sel_agent.c
--- xview-3.2p1.4.orig/lib/libxview/sel/sel_agent.c	2006-03-15 19:56:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/sel/sel_agent.c	2006-03-15 19:57:00.000000000 -0800
@@ -63,6 +63,7 @@
 static void     send_SelectionNotify();
 static void     tvdiff();
 static void     block();
+static int      waitforReadableTimeout();
 static int      is_blocking_over();
 static Seln_result seln_do_request_from_file();
 static void     selection_agent_do_function();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/selection/sel_own.c xview-3.2p1.4/lib/libxview/selection/sel_own.c
--- xview-3.2p1.4.orig/lib/libxview/selection/sel_own.c	2005-01-28 13:29:14.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/selection/sel_own.c	2006-03-15 19:57:00.000000000 -0800
@@ -28,6 +28,7 @@
 Xv_private Time xv_sel_get_last_event_time();
 Pkg_private Sel_owner_info  *xv_sel_find_selection_data();
 Pkg_private Sel_owner_info  *xv_sel_set_selection_data();
+static int SendIncr();
 
 static int DoConversion();
 static int OwnerHandleReply();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/selection/sel_req.c xview-3.2p1.4/lib/libxview/selection/sel_req.c
--- xview-3.2p1.4.orig/lib/libxview/selection/sel_req.c	2005-01-28 13:36:43.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/selection/sel_req.c	2006-03-15 19:57:00.000000000 -0800
@@ -26,7 +26,7 @@
 
 
 Pkg_private char *xv_sel_atom_to_str(/* display, atom */);
-Pkg_private XID SelGetOwnerXID(/* selReq */);
+static XID SelGetOwnerXID(/* selReq */);
 Pkg_private Atom xv_sel_str_to_atom(/* display, string */);
 Pkg_private int xv_sel_add_prop_notify_mask();
 Pkg_private Atom xv_sel_get_property();
@@ -49,13 +49,16 @@
 static int HandleLocalProcess();
 static int LocalMultipleTrans();
 static int ProcessIncr();
+static int XvGetRequestedValue();
+static int ProcessNonBlkIncr();
 static int TransferData();
 static int HandleLocalIncr();
 static int ProcessMultiple();
 static int GetSelection();
 static int ProcessReply();
 static Requestor *SelGetReq();
-
+static int ProcessReq();
+static int OldPkgIsOwner();
 
 /*ARGSUSED*/
 Pkg_private int
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/selection/sel_util.c xview-3.2p1.4/lib/libxview/selection/sel_util.c
--- xview-3.2p1.4.orig/lib/libxview/selection/sel_util.c	2005-01-28 13:29:14.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/selection/sel_util.c	2006-03-15 19:57:00.000000000 -0800
@@ -23,6 +23,7 @@
 static void FreeMultiProp();
 static int	SelMatchReply();
 static Sel_req_tbl *SelMatchReqTbl();
+static int SelFindReply();
 
 Pkg_private struct timeval *
 xv_sel_cvt_xtime_to_timeval( XTime )
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/server/server.c xview-3.2p1.4/lib/libxview/server/server.c
--- xview-3.2p1.4.orig/lib/libxview/server/server.c	2005-01-28 13:35:24.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/server/server.c	2006-03-15 19:57:00.000000000 -0800
@@ -70,6 +70,7 @@
 static unsigned int 	 string_to_modmask();
 static Server_atom_type  save_atom();
 static void 		 server_yield_modifiers();
+static int xv_set_scheduler();
 
 Xv_private char		*xv_strtok();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/ei_text.c xview-3.2p1.4/lib/libxview/textsw/ei_text.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/ei_text.c	2005-01-28 13:34:51.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/ei_text.c	2006-03-15 19:57:00.000000000 -0800
@@ -126,6 +126,7 @@
 #endif
 static struct ei_span_result ei_plain_text_span_of_group();
 static struct ei_process_result ei_plain_text_expand();
+static paint_batch();
 
 struct ei_ops   ei_plain_text_ops = {
     ei_plain_text_destroy,
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/es_file.c xview-3.2p1.4/lib/libxview/textsw/es_file.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/es_file.c	2006-03-15 19:56:45.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/es_file.c	2006-03-15 19:57:00.000000000 -0800
@@ -495,7 +495,7 @@
 }
 
 /* ARGSUSED */
-static          caddr_t
+extern          caddr_t
 #ifdef ANSI_FUNC_PROTO
 es_file_get(Es_handle esh, Es_attribute attribute, ...)
 #else
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/es_mem.c xview-3.2p1.4/lib/libxview/textsw/es_mem.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/es_mem.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/es_mem.c	2006-03-15 19:57:00.000000000 -0800
@@ -116,7 +116,7 @@
 }
 
 /* ARGSUSED */
-static          caddr_t
+extern          caddr_t
 #ifdef ANSI_FUNC_PROTO
 es_mem_get(Es_handle esh, Es_attribute attribute, ...)
 #else
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/ev_display.c xview-3.2p1.4/lib/libxview/textsw/ev_display.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/ev_display.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/ev_display.c	2006-03-15 19:57:00.000000000 -0800
@@ -42,6 +42,7 @@
 Pkg_private Es_index ev_line_start();
 Pkg_private Op_bdry_handle ev_add_op_bdry();
 Pkg_private void ev_extend_to_view_bottom();
+static void     ev_swap_line_table();
 
 Pkg_private Es_index ev_index_for_line();
 Pkg_private struct ei_process_result ev_display_internal();
@@ -940,7 +941,6 @@
 	register int    i;
 	Es_index        pos, pos_to_remember;
 	Pkg_private void ev_lt_format();
-	static void     ev_swap_line_table();
 
 	esbuf.esh = chain->esh;
 	esbuf.buf = buf;
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/ev_op_bdry.c xview-3.2p1.4/lib/libxview/textsw/ev_op_bdry.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/ev_op_bdry.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/ev_op_bdry.c	2006-03-15 19:57:00.000000000 -0800
@@ -25,6 +25,8 @@
 Pkg_private Es_index ev_position_for_physical_line();
 
 Pkg_private void ev_clear_from_margins();
+static int ev_find_finger_internal();
+static ev_remove_finger_internal();
 static Ev_finger_handle ev_insert_finger();
 static Ev_mark_object last_generated_id;
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/ps_impl.c xview-3.2p1.4/lib/libxview/textsw/ps_impl.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/ps_impl.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/ps_impl.c	2006-03-15 19:57:00.000000000 -0800
@@ -61,6 +61,8 @@
 static Es_index ps_read(), ps_scratch_read();
 static Es_index ps_replace(), ps_scratch_replace();
 static int      ps_set();
+static          copy_pieces();
+static int      get_current_offset();
 
 static Es_index write_header_etc();
 
@@ -1575,7 +1577,7 @@
     private->last_write_plus_one = ES_INFINITY;
 }
 
-static          caddr_t
+extern          caddr_t
 #ifdef ANSI_FUNC_PROTO
 ps_get(Es_handle esh, Es_attribute attribute, ...)
 #else
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_again.c xview-3.2p1.4/lib/libxview/textsw/txt_again.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_again.c	2005-01-28 13:34:42.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_again.c	2006-03-15 19:57:00.000000000 -0800
@@ -26,6 +26,7 @@
 
 Pkg_private Es_index textsw_do_input();
 Pkg_private Es_index textsw_do_pending_delete();
+static int textsw_string_min_free();
 
 string_t        null_string = {0, 0, 0};
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_e_menu.c xview-3.2p1.4/lib/libxview/textsw/txt_e_menu.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_e_menu.c	2005-01-28 13:34:49.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_e_menu.c	2006-03-15 19:57:00.000000000 -0800
@@ -75,6 +75,7 @@
 static	int      free_argv();
 static	int      Nargs;
 static	char	*check_filename_locale();
+static           any_shell_meta_e_menu();
 
 extern int      EXTRASMENU_FILENAME_KEY;
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_field.c xview-3.2p1.4/lib/libxview/textsw/txt_field.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_field.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_field.c	2006-03-15 19:57:00.000000000 -0800
@@ -121,7 +121,6 @@
 		    CHAR            marker2[3];
 		    int             marker2_len;
 		    unsigned        direction;
-		    static void     textsw_get_match_symbol();
 
 		    buf = buf + (buf_len - marker1_len);
 		    (void) textsw_get_match_symbol(marker1, marker1_len,
@@ -152,7 +151,6 @@
 		    CHAR            marker2[3];
 		    int             marker2_len;
 		    unsigned        direction;
-		    static void     textsw_get_match_symbol();
 
 		    (void) textsw_get_match_symbol(marker1, marker1_len,
 					 marker2, &marker2_len, &direction);
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_filter.c xview-3.2p1.4/lib/libxview/textsw/txt_filter.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_filter.c	2005-01-28 13:34:55.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_filter.c	2006-03-15 19:57:00.000000000 -0800
@@ -76,6 +76,8 @@
 Pkg_private Es_index textsw_do_input();
 static short unsigned type_for_filter_rec();
 static int      event_code_for_filter_rec();
+static int      talk_to_filter();
+static int      start_filter();
 
 /*
  * WARNING: this is a hack to force the variable to be in memory. this var
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_input.c xview-3.2p1.4/lib/libxview/textsw/txt_input.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_input.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_input.c	2006-03-15 19:57:00.000000000 -0800
@@ -56,6 +56,20 @@
 #ifdef OW_I18N
 Pkg_private     void textsw_implicit_commit_doit();
 #endif
+#ifdef OW_I18N
+Pkg_private int
+#else
+static int
+#endif
+textsw_do_newline();
+static int      textsw_scroll_event();
+static int      textsw_function_key_event();
+static int      textsw_mouse_event();
+static int      textsw_edit_function_key_event();
+static int      textsw_caret_motion_event();
+static int      textsw_field_event();
+static int      textsw_file_operation();
+static int      textsw_erase_action();
 
 #define SPACE_CHAR 0x20
 
@@ -200,14 +214,6 @@
     Notify_arg      arg;
 {
     Pkg_private void     textsw_update_scrollbars();
-    static int      textsw_scroll_event();
-    static int      textsw_function_key_event();
-    static int      textsw_mouse_event();
-    static int      textsw_edit_function_key_event();
-    static int      textsw_caret_motion_event();
-    static int      textsw_field_event();
-    static int      textsw_file_operation();
-    static int      textsw_erase_action();
 
     int             caret_was_up;
     int             result = TEXTSW_PE_USED;
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_menu.c xview-3.2p1.4/lib/libxview/textsw/txt_menu.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_menu.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_menu.c	2006-03-15 19:57:00.000000000 -0800
@@ -47,6 +47,9 @@
 
 Pkg_private Es_index ev_index_for_line();
 static void textsw_done_menu();
+static int  textsw_edit_do_menu_action(),
+            textsw_view_do_menu_action(),
+            textsw_find_do_menu_action();
 
 #define			MAX_SIZE_MENU_STRING	30
 
@@ -140,8 +143,6 @@
     Frame           frame = xv_get(textsw, WIN_FRAME);
     Menu_item       break_mode_item, undo_cmds_item, find_sel_cmds_item,
                     select_field_cmds_item;
-    static int      textsw_edit_do_menu_action(), textsw_view_do_menu_action(),
-                    textsw_find_do_menu_action();
     int             index;
     Pkg_private char *textsw_get_extras_filename();
     Pkg_private int textsw_build_extras_menu_items();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_once.c xview-3.2p1.4/lib/libxview/textsw/txt_once.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_once.c	2005-01-28 13:34:46.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_once.c	2006-03-15 19:57:00.000000000 -0800
@@ -1357,7 +1357,7 @@
 	    
 	xv_destroy(view->drop_site);
 	textsw_view_cleanup(view);
-	VIEW_PRIVATE(view_public) = NULL; /* WG Mar '95 */
+	view = NULL; /* WG Mar '95 */
 	break;
 
       default:			/* Conservative in face of new cases. */
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_popup.c xview-3.2p1.4/lib/libxview/textsw/txt_popup.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_popup.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_popup.c	2006-03-15 19:57:00.000000000 -0800
@@ -104,6 +104,9 @@
 Pkg_private     open_cmd_proc();
 Pkg_private     include_cmd_proc();
 Pkg_private     save_cmd_proc();
+static void     add_exten_item();
+static void     show_dot_files_proc();
+static int      fc_exten_func();
 
 static Notify_value
 textsw_popup_destroy_func(client, status)
@@ -135,7 +138,6 @@
     Frame           popup_frame, base_frame;
     Panel           panel = NULL;
     char           *label;
-    void            add_exten_item();
 #ifdef OW_I18N
     int		    win_use_im = ((popup_type != TEXTSW_MENU_SEL_MARK_TEXT) &&
 				  (popup_type != TEXTSW_MENU_NORMALIZE_LINE));
@@ -502,8 +504,6 @@
     int frame_width;
     int frame_height;
     Panel_item item;
-    void show_dot_files_proc();
-    int  fc_exten_func();
  
     
     panel = xv_get(fc, FRAME_CMD_PANEL);
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_putkey.c xview-3.2p1.4/lib/libxview/textsw/txt_putkey.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_putkey.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_putkey.c	2006-03-15 19:57:00.000000000 -0800
@@ -26,6 +26,7 @@
 Pkg_private Es_handle textsw_esh_for_span();
 Pkg_private Seln_rank textsw_acquire_seln();
 Pkg_private caddr_t  textsw_checkpoint_undo();
+static int textsw_do_put();
 
 Pkg_private int
 textsw_begin_put(view, inform_svc)
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/textsw/txt_selsvc.c xview-3.2p1.4/lib/libxview/textsw/txt_selsvc.c
--- xview-3.2p1.4.orig/lib/libxview/textsw/txt_selsvc.c	2005-01-28 13:28:40.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/textsw/txt_selsvc.c	2006-03-15 19:57:00.000000000 -0800
@@ -28,6 +28,7 @@
 
 Pkg_private Es_status es_copy();
 static Seln_result textsw_seln_yield();
+static int textsw_should_ask_seln_svc();
 
 static char           *shelf_name = "/tmp/textsw_shelf";
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/ttysw/ttyansi.c xview-3.2p1.4/lib/libxview/ttysw/ttyansi.c
--- xview-3.2p1.4.orig/lib/libxview/ttysw/ttyansi.c	2005-01-28 13:37:17.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/ttysw/ttyansi.c	2006-03-15 19:57:00.000000000 -0800
@@ -34,6 +34,9 @@
 
 char           *textsw_checkpoint_undo();
 Textsw_index    textsw_replace_i18n(), textsw_erase_i18n();
+static int      send_input_to_textsw();
+static int      ansi_lf();
+static int      ansi_char();
 
 /*
  * jcb	-- remove continual cursor repaint in shelltool windows also known to
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/ttysw/tty_init.c xview-3.2p1.4/lib/libxview/ttysw/tty_init.c
--- xview-3.2p1.4.orig/lib/libxview/ttysw/tty_init.c	2005-01-28 13:37:20.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/ttysw/tty_init.c	2006-03-15 19:57:00.000000000 -0800
@@ -98,6 +98,7 @@
 extern long     lseek();
 #endif
 char           *textsw_checkpoint_undo();
+static int      ttyinit();
 
 /* static */ void ttysw_parseargs();
 
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/ttysw/tty_main.c xview-3.2p1.4/lib/libxview/ttysw/tty_main.c
--- xview-3.2p1.4.orig/lib/libxview/ttysw/tty_main.c	2005-01-28 13:37:14.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/ttysw/tty_main.c	2006-03-15 19:57:00.000000000 -0800
@@ -84,6 +84,10 @@
 #ifdef OW_I18N
 extern Textsw_index textsw_insert_wcs();
 #endif
+static int ttysw_process_point();
+static int ttysw_process_adjust();
+static int ttysw_process_motion();
+static int ttysw_process_keyboard();
 
 /*
  * jcb	-- remove continual cursor repaint in shelltool windows also known to
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/ttysw/ttyselect.c xview-3.2p1.4/lib/libxview/ttysw/ttyselect.c
--- xview-3.2p1.4.orig/lib/libxview/ttysw/ttyselect.c	2005-01-28 13:29:22.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/ttysw/ttyselect.c	2006-03-15 19:57:00.000000000 -0800
@@ -70,7 +70,8 @@
 /* static */ void ttygetselection();	/* BUG ALERT: No XView prefix */
 
 static int      ttysel_insel(),
-                ttysel_eq();
+                ttysel_eq(),
+                ttysel_resynch();
 
 static Seln_result ttysel_copy_in(),
                 ttysel_copy_out(),
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/win/win_input.c xview-3.2p1.4/lib/libxview/win/win_input.c
--- xview-3.2p1.4.orig/lib/libxview/win/win_input.c	2005-01-28 13:38:04.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/win/win_input.c	2006-03-15 19:57:00.000000000 -0800
@@ -61,6 +61,7 @@
 static int      GetButtonEvent();
 static int      win_translate_KP_keysym();
 static int 	translate_key();
+static int  xevent_to_event();
 
 extern struct rectlist *win_get_damage();
 extern void     server_set_timestamp();
diff -urN -x 'Make*' -x '*html' -x '*man' -x make -x '*imake*' -x parse.c -x olvwmrc.c xview-3.2p1.4.orig/lib/libxview/window/window_set.c xview-3.2p1.4/lib/libxview/window/window_set.c
--- xview-3.2p1.4.orig/lib/libxview/window/window_set.c	2005-01-28 13:28:51.000000000 -0800
+++ xview-3.2p1.4/lib/libxview/window/window_set.c	2006-03-15 19:57:00.000000000 -0800
@@ -40,6 +40,7 @@
 static 		void	set_mask_bit(),
 			unset_mask_bit(),
 			window_set_event_mask(),
+			window_set_softkey_labels(),
 			window_grab_selectbutton(),
     			window_ungrab_selectbutton(),
 			win_map(),
@@ -1072,7 +1073,6 @@
 
       case WIN_SOFT_FNKEY_LABELS: {
         register Xv_Drawable_info 	*info;
-        static void                   	 window_set_softkey_labels();
 
         DRAWABLE_INFO_MACRO(win_public, info);
         win->softkey_flag = TRUE;
