diff -Nu -Nu -uarN asgl-1.3.2.orig/scripts/Makefile.include1 asgl-1.3.2/scripts/Makefile.include1
--- asgl-1.3.2.orig/scripts/Makefile.include1	2006-11-10 00:16:11.000000000 +0100
+++ asgl-1.3.2/scripts/Makefile.include1	2008-01-29 16:08:42.000000000 +0100
@@ -348,15 +348,15 @@
 	-if [ `echo ${LIBFILES} | wc -w` -gt 0 ] ; then cp -r ${LIBFILES} ${LIBDIR}; fi
 	-cp ${PROGRAM}_${ASGL_EXECUTABLE_TYPE} ${BINDIR}/${PROGRAM}_${ASGL_EXECUTABLE_TYPE}; \
 	strip ${BINDIR}/${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
-	sed -e "s;BIN_value;${BINDIR};" \
-		-e "s;LIB_value;${LIBDIR};" \
+	sed -e "s;BIN_value;/usr/lib/libasgl;" \
+		-e "s;LIB_value;/usr/lib/libasgl/libasgl;" \
 		asgl.script > ${BINDIR}/${EXECSCRIPT}
-	chmod a+rwx ${BINDIR}
-	chmod a+rwx ${LIBDIR}
-	chmod a+rw ${BINDIR}/*
-	chmod a+rw ${LIBDIR}/*
-	chmod a+rwx ${BINDIR}/${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
-	chmod a+rwx ${BINDIR}/${EXECSCRIPT}
+	chmod 755 ${BINDIR}
+	chmod 755 ${LIBDIR}
+	chmod a+r ${BINDIR}/*
+	chmod a+r ${LIBDIR}/*
+	chmod a+rx ${BINDIR}/${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	chmod a+rx ${BINDIR}/${EXECSCRIPT}
 
 
 # Select the right compilation flags given COMPILER selection. The whole 
