--- validation-v8.061-prod-src/etc/platform.sh	2004-08-20 17:26:48.000000000 +0200
+++ validation-v8.061-prod-src/etc/platform.sh.new	2008-12-28 17:30:17.000000000 +0100
@@ -39,23 +39,33 @@
     ;;
 
     Linux)
-#     Check if it is GCC version 3.x
-      gcc_ver=`gcc --version | grep -e " 3\."`
+#     Check if it is GCC version 3.x#   Check if it is a Linux platform
+#    Linux)
+#     Check if it is GCC version 4.x
+      gcc_ver=`gcc --version | grep -e " 4\."`
       if [[ -z $gcc_ver ]]
       then
-#     It is not GCC version 3.x. Check if it is GCC version 2.x
-        gcc_ver=`gcc --version | grep -e "2\."`
+#       It is not GCC version 4.x. Check if it is GCC version 3.x
+        gcc_ver=`gcc --version | grep -e " 3\."`
         if [[ -z $gcc_ver ]]
         then
-#         It is not GCC version 2.x either. Production can not be compiled.
-          sysid="unknown"
+#       It is not GCC version 3.x. Check if it is GCC version 2.x
+          gcc_ver=`gcc --version | grep -e "2\."`
+          if [[ -z $gcc_ver ]]
+          then
+#           It is not GCC version 2.x either. Production can not be compiled.
+            sysid="unknown"
+          else
+#           It is GCC version 2.x
+            sysid="gnu2"
+          fi
         else
-#         It is GCC version 2.x
-          sysid="gnu2"
+#         It is GCC version 3.x
+          sysid="gnu3"
         fi
       else
-#       It is GCC version 3.x
-        sysid="gnu3"
+#         It is GCC version 4.x
+          sysid="gnu4"
       fi
     ;;
 
