--- validation-v8.061-prod-src/Makefile	2004-08-20 17:26:46.000000000 +0200
+++ validation-v8.061-prod-src/Makefile.new	2009-01-26 15:25:23.518702164 +0100
@@ -20,9 +20,8 @@
 
 DATADIRS = data
 
-LIBDIRS =  misclib-v2.1 regex-v2.1 ciflib-common-v3.02 cifobj-common-v4.04 \
-           btree-obj-v3.1 cif-table-obj-v6.1 cifparse-obj-v6.3 \
-           connect-v2.2 filterlib-v8.1 htmllib-v3.1 validation-v6.02
+LIBDIRS =  cifobj-common-v4.04 btree-obj-v3.1 filterlib-v8.1 htmllib-v3.1 validation-v6.02
+
 all:	compile
 
 test:   compile
