diff --git a/knockout_modeler/ko.py b/knockout_modeler/ko.py
index 5028470..5a7f2fb 100644
--- a/knockout_modeler/ko.py
+++ b/knockout_modeler/ko.py
@@ -1,6 +1,5 @@
 from django.db import models
 from django.template.loader import render_to_string
-from django_fake_model import models as fake_models
 
 import cgi
 try:
@@ -46,7 +45,7 @@ def get_object_data(obj, fields, safe):
 
             try:
                 attribute = getattr(obj, str(field))
-                if isinstance(attribute, dict) or isinstance(attribute, models.Model) or isinstance(attribute, fake_models.FakeModel):
+                if isinstance(attribute, dict) or isinstance(attribute, models.Model):
                     attribute_fields = get_fields(attribute)
                     object_data = get_object_data(attribute, attribute_fields, safe) # Recur
                     temp_dict[field] = object_data
