--- pysize/main.py	2007-03-11 16:09:53.000000000 +0100
+++ pysize/main.py.new	2009-01-01 23:59:32.351548476 +0100
@@ -69,14 +69,6 @@
     stats.print_stats(40)
     os.remove(prof_file)
 
-def _try_psyco():
-    try:
-        # Try to use psyco if available
-        import psyco
-        psyco.full()
-    except ImportError:
-        pass
-
 def main():
     install_sigquit_traceback()
     _try_psyco()
