diff --git a/src/main/com/mucommander/commons/file/impl/rar/RarArchiveFile.java b/src/main/com/mucommander/commons/file/impl/rar/RarArchiveFile.java
index f7a7196..1fc5fa7 100644
--- a/src/main/com/mucommander/commons/file/impl/rar/RarArchiveFile.java
+++ b/src/main/com/mucommander/commons/file/impl/rar/RarArchiveFile.java
@@ -30,8 +30,8 @@ import com.mucommander.commons.file.ArchiveEntryIterator;
 import com.mucommander.commons.file.UnsupportedFileOperationException;
 import com.mucommander.commons.file.WrapperArchiveEntryIterator;
 
-import de.innosystec.unrar.exception.RarException;
-import de.innosystec.unrar.rarfile.FileHeader;
+import com.github.junrar.exception.RarException;
+import com.github.junrar.rarfile.FileHeader;
 
 /**
  * RarArchiveFile provides read-only access to archives in the Rar format.
@@ -80,7 +80,7 @@ public class RarArchiveFile extends AbstractROArchiveFile {
     }
     
     /**
-     * Creates and return an {@link ArchiveEntry()} whose attributes are fetched from the given {@link com.mucommander.commons.file.impl.rar.provider.de.innosystec.unrar.rarfile.FileHeader}
+     * Creates and return an {@link ArchiveEntry()} whose attributes are fetched from the given {@link com.mucommander.commons.file.impl.rar.provider.com.github.junrar.rarfile.FileHeader}
      *
      * @param header the object that serves to initialize the attributes of the returned ArchiveEntry
      * @return an ArchiveEntry whose attributes are fetched from the given FileHeader
diff --git a/src/main/com/mucommander/commons/file/impl/rar/RarFile.java b/src/main/com/mucommander/commons/file/impl/rar/RarFile.java
index 9e0d225..ea810a1 100644
--- a/src/main/com/mucommander/commons/file/impl/rar/RarFile.java
+++ b/src/main/com/mucommander/commons/file/impl/rar/RarFile.java
@@ -26,10 +26,10 @@ import com.mucommander.commons.file.AbstractFile;
 import com.mucommander.commons.file.UnsupportedFileOperationException;
 import com.mucommander.commons.util.CircularByteBuffer;
 
-import de.innosystec.unrar.Archive;
-import de.innosystec.unrar.exception.RarException;
-import de.innosystec.unrar.exception.RarException.RarExceptionType;
-import de.innosystec.unrar.rarfile.FileHeader;
+import com.github.junrar.Archive;
+import com.github.junrar.exception.RarException;
+import com.github.junrar.exception.RarException.RarExceptionType;
+import com.github.junrar.rarfile.FileHeader;
 
 /**
  * 
