--- arp_warp_7.0.orig/install_csh.sh    2007-11-19 13:33:06.000000000 +0100
+++ arp_warp_7.0/install_csh.sh 2007-11-19 13:33:38.000000000 +0100
@@ -14,7 +14,7 @@
 alias runtesttail   "echo test | tail -1"
 alias runtestrefmac 'refmac5 -i'
 alias runtestpython 'python -V'
-set arpwarphome = `pwd`
+set arpwarphome = "/usr/lib/arp_warp-7.0/"
 #
 # Checking availability of 'tr' and setting excape commands
 runtesttr >& /dev/null
