--- arp_warp_7.0.orig/install_csh.sh	2007-11-19 13:33:06.000000000 +0100
+++ arp_warp_7.0/install_csh.sh	2007-11-19 14:02:10.000000000 +0100
@@ -128,24 +128,24 @@
 endif
 #
 # Checking ARP/wARP tar ball directories
-printnl 'Checking '"${b1}"'ARP/wARP directory structure'"${b0}"' - '
-set dirlist = ( 'bin' 'share' 'byte-code' 'flex-wARP-src' )
-foreach try ( $dirlist )
-  set dirtest = $arpwarphome'/'$try
-  if ( ! -d $dirtest ) then
-    printl "${b1}"$dirtest' directory is missing'
-    printl 'Please untar and unzip the original ARP/wARP 7.0 distribution file'"${b0}"
-    goto installfailed
-  else
-    set dircontent = (`ls $dirtest`)
-    if ( $#dircontent == 0 ) then
-      printl "${b1}"$dirtest' directory is empty'
-      printl 'Please untar and unzip the original ARP/wARP 7.0 distribution file'"${b0}"
-      goto installfailed
-    endif
-  endif
-end
-printl "${b1}"'OK'"${b0}"
+#printnl 'Checking '"${b1}"'ARP/wARP directory structure'"${b0}"' - '
+#set dirlist = ( 'bin' 'share' 'byte-code' 'flex-wARP-src' )
+#foreach try ( $dirlist )
+#  set dirtest = $arpwarphome'/'$try
+#  if ( ! -d $dirtest ) then
+#    printl "${b1}"$dirtest' directory is missing'
+#    printl 'Please untar and unzip the original ARP/wARP 7.0 distribution file'"${b0}"
+#    goto installfailed
+#  else
+#    set dircontent = (`ls $dirtest`)
+#    if ( $#dircontent == 0 ) then
+#      printl "${b1}"$dirtest' directory is empty'
+#      printl 'Please untar and unzip the original ARP/wARP 7.0 distribution file'"${b0}"
+#      goto installfailed
+#    endif
+#  endif
+#end
+#printl "${b1}"'OK'"${b0}"
 #
 # Checking availability of 'python', must be 2.2 or higher
 printnl 'Checking '"${b1}"'python'"${b0}"' installation          - '
