# fix compilation error (strlen not part of std namespace)

--- a/rosetta++/dna.cc	2008-04-17 22:27:01.000000000 +0200
+++ b/rosetta++/dna.cc	2008-12-05 23:18:59.000000000 +0100
@@ -54,6 +54,7 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <cstring>
 
 ////////////////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
