# fix gcc-3.3.6 compiler warning (const static vs. static const)
--- a/rosetta++/atom_chem.h	2008-04-17 22:27:00.000000000 +0200
+++ b/rosetta++/atom_chem.h	2008-12-12 00:13:35.000000000 +0100
@@ -49,7 +49,7 @@
 
 
 		// array of two character symbols for element types.
-		const static std::string element_type_symbol[]={"Uu", " H", "He", "Li", "Be",
+		static const std::string element_type_symbol[]={"Uu", " H", "He", "Li", "Be",
 			" B", " C", " N", " O", " F", "Ne", "Na", "Mg", "Al",
 			"Si", " P", " S", "Cl", "Ar", " K", "Ca", "Sc", "Ti",
 			" V", "Cr", "Mn", "Fe", "Co", "Ni", "Cu", "Zn", "Ga",
@@ -85,7 +85,7 @@
                         DOUBLE_BONDED_SULFUR, DOUBLE_BONDED_SULFUR_W_ONE_OXYGEN, SULPHONIC_SULFUR,
                         FLUORINE_ATOM, CHLORINE_ATOM, BROMINE_ATOM, IODINE_ATOM };
 
-		const static std::string ligandatomtype_string[] = {"UNDEFINED_ATOM_TYPE", "QUATERNARY_CARBON", "TERTIARY_CARBON",
+		static const std::string ligandatomtype_string[] = {"UNDEFINED_ATOM_TYPE", "QUATERNARY_CARBON", "TERTIARY_CARBON",
 			"SECONDARY_CARBON", "PRIMARY_CARBON", "ALKENE_CARBON",
 			"ALKENE_CARBON_W_HYDROGEN", "ALKYNE_CARBON", "AROMATIC_CARBON",
 			"AROMATIC_CARBON_W_HYDROGEN", "TERTIARY_NITROGEN", "SECONDARY_NITROGEN",
