# fix scons deprecated warning about environment.Copy()

--- a/rosetta++/SConscript.src.rosetta++	2008-04-17 22:27:00.000000000 +0200
+++ b/rosetta++/SConscript.src.rosetta++	2008-12-05 02:01:35.000000000 +0100
@@ -87,7 +87,7 @@
 
 # Transform the modified settings into SCons Environment variables.
 # Gives priority to project settings over global settings.
-env = build.environment.Copy()
+env = build.environment.Clone()
 env.Prepend(**actual.symbols())
 
 # The build target's base name is "rosetta", not "rosetta++"
