diff --git a/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/beans/CollectingReaderEventListener.java b/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/beans/CollectingReaderEventListener.java
index b581a05..4817bc5 100644
--- a/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/beans/CollectingReaderEventListener.java
+++ b/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/beans/CollectingReaderEventListener.java
@@ -38,9 +38,9 @@ public class CollectingReaderEventListener implements ReaderEventListener {
 
 	private final List defaults = new LinkedList();
 
-	private final Map componentDefinitions = new LinkedHashMap<>(8);
+	private final Map componentDefinitions = new LinkedHashMap<Object, Object>(8);
 
-	private final Map aliasMap = new LinkedHashMap<>(8);
+	private final Map aliasMap = new LinkedHashMap<Object, Object>(8);
 
 	private final List imports = new LinkedList();
 
diff --git a/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/sample/beans/TestBean.java b/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/sample/beans/TestBean.java
index cb276da..3481c6a 100644
--- a/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/sample/beans/TestBean.java
+++ b/spring-framework-3.2.4.RELEASE/spring-beans/src/test/java/org/springframework/tests/sample/beans/TestBean.java
@@ -76,13 +76,13 @@ public class TestBean implements BeanNameAware, BeanFactoryAware, ITestBean, IOt
 
 	private Float myFloat = new Float(0.0);
 
-	private Collection<? super Object> friends = new LinkedList<>();
+	private Collection<? super Object> friends = new LinkedList<Object>();
 
-	private Set<?> someSet = new HashSet<>();
+	private Set<?> someSet = new HashSet<Object>();
 
-	private Map<?, ?> someMap = new HashMap<>();
+	private Map<?, ?> someMap = new HashMap<Object, Object>();
 
-	private List<?> someList = new ArrayList<>();
+	private List<?> someList = new ArrayList<Object>();
 
 	private Properties someProperties = new Properties();
 
