--- ccp4-6.0.2.orig/src/Makefile.in     2008-03-02 17:59:15.000000000 +0100
+++ ccp4-6.0.2/src/Makefile.in  2008-03-02 18:01:35.000000000 +0100
@@ -45,7 +45,7 @@
 OTHERS = rsps anglen secstr proclean pplot gfac2pdb tplot mplot probplot rmsdev nb \
          scala dm dmmulti stereo mapmask maprot molrep ncsmask rdent restrain \
          sfcheck sftools tlsanl tlsextract topp toplist dyndom sc combat mlphare fffear ffjoin \
-         anisoanl refmac5 libcheck beast dtrek2scala
+         anisoanl beast dtrek2scala
 
 # targets built from .c sources:
 CSIMPLE = pltdev binsort