diff --git a/scripts/Makefile.include1 b/scripts/Makefile.include1
index 0b481ca..95ebac5 100644
--- a/scripts/Makefile.include1
+++ b/scripts/Makefile.include1
@@ -26,12 +26,12 @@ CHECK_OPTS=-array=0 -calltree -declare -portability -novice -nopretty \
 # F2C
 .F.m88k_o:
 	/lib/cpp -P ${INCLUDE} ${DDEFS} $*.F > $*.f
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 	rm $*.f
 
 .F.f2c_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 #	/lib/cpp -P ${INCLUDE} ${DDEFS} $*.F > $*.f
 #	f2c ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
@@ -53,91 +53,91 @@ CHECK_OPTS=-array=0 -calltree -declare -portability -novice -nopretty \
 
 # F2C
 .F.next_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F
-#	${FC} ${INCLUDE} ${DDEFS} -DSMALL ${FFLAGS} -c $*.F
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F
+#	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} -DSMALL ${FFLAGS} -c $*.F
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 
 .F.sun4_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o ${*F}.o
-#	${FC} ${INCLUDE} ${DDEFS} -DSMALL ${FFLAGS} -c $*.F -o ${*F}.o
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o ${*F}.o
+#	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} -DSMALL ${FFLAGS} -c $*.F -o ${*F}.o
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 
 .F.sol2_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o ${*F}.o
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o ${*F}.o
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 
 .F.sol2f90_o:
 	/usr/ccs/lib/cpp -P ${INCLUDE} ${DDEFS} -Df90 $*.F > $*.f
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 	rm $*.f
 
 # F2C:
 .F.sun4-5.3_o:
 	cpp -P ${INCLUDE} ${DDEFS} $*.F > $*.f
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 	rm $*.f
 
 .F.hp9000s700_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.cray_o:
-	${FC} ${INCLUDE} ${DDEFS} -Dcray ${FFLAGS} -c $*.F
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} -Dcray ${FFLAGS} -c $*.F
 	@if [ -f ${*F}.o -a ${*F}.o != $@  ] ; then mv ${*F}.o $@ ; fi
 
 .F.alpha_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.alphaf90_o:
-	${FC} ${INCLUDE} ${DDEFS} -DALPHA ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} -DALPHA ${FFLAGS} -c $*.F -o $@
 
 .F.decstation_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.iris4d_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.r10000_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.f90_o:
-	${FC} ${INCLUDE} -Df90 ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} -Df90 ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 # For debugging on IRIS4D:
 .F.iris4dt_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.g77_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.gfortran_o:
-	${FC} ${INCLUDE} -Df90 -Df2c ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} -Df90 -Df2c ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.i386-intel_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.mac10v4-xlf_o:
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.convex_o:
 	ln $*.F $*.f
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.f -o $@
 	rm $*.f
 
 # It seems that R8000 f77 compiler cannot do both CPP and F77 at the same time
 # anymore:
 .F.r8000_o:
-	${FC} ${INCLUDE} ${DDEFS} -P $*.F
-	${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c ${*F}.i -o $@
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} -P $*.F
+	${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c ${*F}.i -o $@
 	rm ${*F}.i
 
 # .F.r8000_o:
-# ${FC} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
+# ${FC} ${FCFLAGS} ${INCLUDE} ${DDEFS} ${FFLAGS} -c $*.F -o $@
 
 .F.rs6000.o:
 	/lib/cpp -P ${INCLUDE} -DAIX ${DDEFS} $*.F > $*.f
-	${FC} ${FFLAGS} -c $*.f -o $@
+	${FC} ${FCFLAGS} ${FFLAGS} -c $*.f -o $@
 	rm $*.f
 
 # $*     ... directory/root
@@ -465,10 +465,10 @@ flags:
 # Linking rules:
 
 ${PROGRAM}_m88k: ${FILES:.F=.m88k_o}
-	${FC} ${FFLAGS} ${FILES:.F=.m88k_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.m88k_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_f2c: ${FILES:.F=.f2c_o}
-	${FC} ${FFLAGS} ${FILES:.F=.f2c_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.f2c_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_i386-linux: ${FILES:.F=.i386-linux_o}
 	f77 ${FILES:.F=.i386-linux_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
@@ -477,67 +477,67 @@ ${PROGRAM}_i386-absoft: ${FILES:.F=.i386-absoft_o}
 	f90 ${FILES:.F=.i386-absoft_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_iris4d: ${FILES:.F=.iris4d_o}
-	${FC} ${FFLAGS} ${FILES:.F=.iris4d_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.iris4d_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_r10000: ${FILES:.F=.r10000_o}
-	${FC} ${FFLAGS} ${FILES:.F=.r10000_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.r10000_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_f90: ${FILES:.F=.f90_o}
-	${FC} ${FFLAGS} ${FILES:.F=.f90_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.f90_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_iris4dt: ${FILES:.F=.iris4dt_o}
-	${FC} ${FFLAGS} ${FILES:.F=.iris4dt_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.iris4dt_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_r8000: ${FILES:.F=.r8000_o}
-	${FC} ${FFLAGS} ${FILES:.F=.r8000_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.r8000_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_g77: ${FILES:.F=.g77_o}
-	${FC} ${FFLAGS} ${FILES:.F=.g77_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.g77_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_gfortran: ${FILES:.F=.gfortran_o}
-	${FC} ${FFLAGS} ${FILES:.F=.gfortran_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.gfortran_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_i386-intel: ${FILES:.F=.i386-intel_o}
-	${FC} ${FFLAGS} ${FILES:.F=.i386-intel_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.i386-intel_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_mac10v4-xlf: ${FILES:.F=.mac10v4-xlf_o}
-	${FC} ${FFLAGS} ${FILES:.F=.mac10v4-xlf_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.mac10v4-xlf_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_next: ${FILES:.F=.next_o}
-	${FC} ${FFLAGS} ${FILES:.F=.next_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.next_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_sun4: ${FILES:.F=.sun4_o}
-	${FC} ${FFLAGS} ${FILES:.F=.sun4_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.sun4_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_sol2: ${FILES:.F=.sol2_o}
-	${FC} ${FFLAGS} ${FILES:.F=.sol2_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.sol2_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_sol2f90: ${FILES:.F=.sol2f90_o}
-	${FC} ${FFLAGS} ${FILES:.F=.sol2f90_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.sol2f90_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_sun4-5.3: ${FILES:.F=.sun4-5.3_o}
-	${FC} ${FFLAGS} ${FILES:.F=.sun4-5.3_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.sun4-5.3_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_alpha: ${FILES:.F=.alpha_o}
-	${FC} ${FFLAGS} ${FILES:.F=.alpha_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.alpha_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_alphaf90: ${FILES:.F=.alphaf90_o}
-	${FC} ${FFLAGS} ${FILES:.F=.alphaf90_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.alphaf90_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_convex: ${FILES:.F=.convex_o}
-	${FC} ${FFLAGS} ${FILES:.F=.convex_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.convex_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_rs6000: ${FILES:.F=.rs6000.o}
-	${FC} ${FFLAGS} ${FILES:.F=.rs6000.o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.rs6000.o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_decstation: ${FILES:.F=.decstation_o}
-	${FC} ${FFLAGS} ${FILES:.F=.decstation_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.decstation_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_cray: ${FILES:.F=.cray_o}
-	${FC} ${FFLAGS} ${FILES:.F=.cray_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.cray_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_hp9000s700: ${FILES:.F=.hp9000s700_o}
-	${FC} ${FFLAGS} ${FILES:.F=.hp9000s700_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.hp9000s700_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
 
 ${PROGRAM}_stardent: ${FILES:.F=.stardent_o}
-	${FC} ${FFLAGS} ${FILES:.F=.stardent_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
+	${FC} ${FCFLAGS} ${FFLAGS} ${FILES:.F=.stardent_o} ${LDFLAGS} -o ${PROGRAM}_${ASGL_EXECUTABLE_TYPE}
