diff -Nu -Nu -uarN imosflm.orig/src/imosflm imosflm/src/imosflm
--- imosflm.orig/src/imosflm	2008-04-15 17:28:31.000000000 +0200
+++ imosflm/src/imosflm	2008-04-15 17:29:30.000000000 +0200
@@ -7,8 +7,8 @@
 #  edit and uncomment the following lines to point to the site-wide wish8.4
 #  and Mosflm 7.0.2 executables
 
-#export MOSFLM_WISH=/path/to/your/wish8.4
-#export MOSFLM_EXEC=/path/to/your/ipmosflm
+export MOSFLM_WISH=/usr/bin/wish8.4
+export MOSFLM_EXEC=/usr/bin/ipmosflm
 STARTDIR=
 IMAGEDIR=
 STARTDIRSET=
