Index: source/com/mucommander/conf/impl/MuConfiguration.java
===================================================================
--- source/com/mucommander/conf/impl/MuConfiguration.java	(revision 2947)
+++ source/com/mucommander/conf/impl/MuConfiguration.java	(working copy)
@@ -88,6 +88,9 @@
     public static final boolean DEFAULT_ENABLE_SYSTEM_NOTIFICATIONS = OsFamilies.MAC_OS_X.isCurrent() ||
             (OsFamilies.WINDOWS.isCurrent() && JavaVersions.JAVA_1_6.isCurrentOrHigher());
 
+    /** Excluded Mountpoints */
+    public static final String EXCLUDED_MOUNTPOINTS 			  = "excludeMountPoints";
+
     // - Shell variables -----------------------------------------------------
     // -----------------------------------------------------------------------
     /** Section describing the shell configuration. */
Index: source/com/mucommander/file/RootFolders.java
===================================================================
--- source/com/mucommander/file/RootFolders.java	(revision 2947)
+++ source/com/mucommander/file/RootFolders.java	(working copy)
@@ -20,6 +20,7 @@
 package com.mucommander.file;
 
 import com.mucommander.Debug;
+import com.mucommander.conf.impl.MuConfiguration;
 import com.mucommander.file.impl.local.LocalFile;
 import com.mucommander.runtime.OsFamilies;
 import com.mucommander.runtime.OsFamily;
@@ -104,6 +105,10 @@
             String line;
             AbstractFile file;
             String folderPath;
+
+            Vector userExcludedMountPoints = MuConfiguration.getListVariable(MuConfiguration.EXCLUDED_MOUNTPOINTS, ",");
+            if (userExcludedMountPoints == null) { userExcludedMountPoints = new Vector(); }
+
             while((line=br.readLine())!=null) {
                 // Skip comments
                 // JS, 2008-09-25: Exclude empty lines too
@@ -113,7 +118,7 @@
                     // path is second token
                     st.nextToken();
                     folderPath = st.nextToken();
-                    if(!(folderPath.equals("/proc") || folderPath.equals("none") || folderPath.equals("/dev/shm"))) {
+                    if(!(folderPath.equals("/proc") || folderPath.equals("none") || folderPath.equals("/dev/shm") || userExcludedMountPoints.contains(folderPath))) {
                         file = FileFactory.getFile(folderPath);
                         if(file!=null && !v.contains(file))
                             v.add(file);
@@ -122,7 +127,7 @@
             }
         }
         catch(Exception e) {
-            if(Debug.ON) Debug.trace("Error reading /etc/fstab entries: "+ e);
+            if(Debug.ON) Debug.trace("Error reading /etc/fstab entries: " + e);
         }
         finally {
             if(br != null) {

