--- makefile	2008-12-03 16:04:06.000000000 +0100
+++ makefile.new	2008-12-03 16:42:45.000000000 +0100
@@ -55,8 +55,8 @@
 	cd CPP/7zip/UI/Console           ; $(MAKE) all
 
 7zG: common7z
-	cd bin ; rm -f Lang ; ln ../GUI/Lang .
-	cd bin ; rm -f help ; ln ../GUI/help .
+	cd bin ; rm -f Lang ; ln -s ../GUI/Lang
+	cd bin ; rm -f help ; ln -s ../GUI/help
 	cd CPP/7zip/UI/GUI               ; $(MAKE) all
 
 clean:
