diff --git a/courier-imap-4.15/libs/maildir/configure.ac b/courier-imap-4.15/libs/maildir/configure.ac
index 01604a6..ebc7fee 100644
--- a/courier-imap-4.15/libs/maildir/configure.ac
+++ b/courier-imap-4.15/libs/maildir/configure.ac
@@ -73,6 +73,12 @@ AC_SYS_LARGEFILE
 dnl Checks for library functions.
 AC_CHECK_HEADER(fam.h, :, :)
 AC_CHECK_FUNCS(symlink readlink strcasecmp utime utimes)
+AC_ARG_WITH(fam, [  --with-fam              Compile with FAM support],
+       fam="$withval",
+       fam="no")
+
+if test "$fam" != "no"
+then
 AC_CHECK_LIB(fam, FAMOpen,  [
 			LIBFAM=-lfam
 			AC_DEFINE_UNQUOTED(HAVE_FAM,1,
@@ -87,6 +93,7 @@ AC_MSG_WARN([[package for your operating system.]])
 AC_MSG_ERROR([[FAM development libraries not found.]]) ]
 					)
 		  ])
+fi
 
 AC_SUBST(LIBFAM)
 
