diff --git a/tokyocabinet-1.4.48/Makefile.in b/tokyocabinet-1.4.48/Makefile.in
index 8846c38..005083a 100644
--- a/tokyocabinet-1.4.48/Makefile.in
+++ b/tokyocabinet-1.4.48/Makefile.in
@@ -54,8 +54,8 @@ CFLAGS = @MYCFLAGS@
 LDFLAGS = @MYLDFLAGS@
 CMDLDFLAGS = @MYCMDLDFLAGS@
 LIBS = @LIBS@
-LDENV = LD_RUN_PATH=/lib:/usr/lib:$(LIBDIR):$(HOME)/lib:/usr/local/lib:@MYRUNPATH@:.
-RUNENV = @MYLDLIBPATHENV@=.:/lib:/usr/lib:$(LIBDIR):$(HOME)/lib:/usr/local/lib:@MYRUNPATH@
+LDENV = LD_RUN_PATH=/lib:/usr/lib:$(LIBDIR)
+RUNENV = @MYLDLIBPATHENV@=.:/lib:/usr/lib:$(LIBDIR)
 POSTCMD = @MYPOSTCMD@
 
 
