--- instlib/source/Makefile.proto	2000-09-07 20:35:33.000000000 +0200
+++ instlib/source/Makefile.proto.new	2009-07-03 12:16:36.000000000 +0200
@@ -42,9 +42,8 @@
 ../bin/cns_solve: $(DEPENDS) 
 	@ echo ; echo "linking: cns_solve"; echo
 	@ cns_string=`date "+%y%m%d%H%M"`; \
-	  $(LD) -o cns_solve-$$cns_string.exe $(OBJS) dmemory.o machine_c.o \
-	                                      $(LDFLAGS) \
-	                                      $(CNS_FFTDIR) $(CNS_FFTLIB); \
+	  $(LD) $(LDFLAGS) -o cns_solve-$$cns_string.exe $(OBJS) dmemory.o machine_c.o \
+	                                      $(CNS_FFTDIR) $(CNS_FFTLIB) $(OMP_LIB); \
 	  if [ -x cns_solve-$$cns_string.exe ]; \
 	    then echo "created executable file cns_solve-$$cns_string.exe"; \
 	         echo ""; cd ../bin; $(RM) -f cns_solve; $(RM) -f cns; \
