 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index f4db86d..7d8bb33 100644
--- a/Makefile
+++ b/Makefile
@@ -116,10 +116,10 @@ subdirs:
 	fi
 
 subagent32: standardall $(LOBJS)
-	net-snmp-config --compile-subagent arecamib --cflags "-I. -I.. -I../../includes /usr/local/lib/libarclib32.so" $(OBJS)
+	net-snmp-config --compile-subagent arecamib --cflags "-I. -I.. -I../../includes linux/i386/libarclib32.so -lstdc++" $(OBJS)
 	
 subagent64: standardall $(LOBJS)
-	net-snmp-config --compile-subagent arecamib --cflags "-I. -I.. -I../../includes /usr/local/lib/libarclib64.so" $(OBJS)
+	net-snmp-config --compile-subagent arecamib --cflags "-I. -I.. -I../../includes linux/x86-64/libarclib64.so" --ldflags "$(LDFLAGS) -lstdc++" $(OBJS)
 
 # cleaning targets
 #
