--- cctbx_sources/libtbx/env_config.py	2009-01-19 08:54:57.000000000 +0100
+++ cctbx_sources/libtbx/env_config.py.new	2009-03-04 12:15:08.000000000 +0100
@@ -891,8 +891,8 @@
     else:
       action = self.write_bin_sh_dispatcher
       ext = ""
-      try: os.chmod(source_file, 0755)
-      except OSError: pass
+      #try: os.chmod(source_file, 0755)
+      #except OSError: pass
     target_file_ext = target_file + ext
     remove_or_rename(target_file_ext)
     try: action(source_file, target_file_ext)
