diff -uarN src.orig/pathManager.cpp src/pathManager.cpp
--- src.orig/pathManager.cpp	2008-01-12 21:20:44.000000000 +0100
+++ src/pathManager.cpp	2008-01-12 21:21:17.000000000 +0100
@@ -41,7 +41,7 @@
 	if(homeDir.Last() != '/')
 		homeDir += wxT("/");
 
-	mImgPath = wxString::Format(wxT("%s/pixmaps/fahmon/"), wxT(DATADIR));
+	mImgPath = wxString::Format(wxT("/usr/share/fahmon/images/"), wxT(DATADIR));
 	mCfgPath = homeDir + wxT(".fahmon/");
 	mXYZPath = homeDir + wxT(".fahmon/xyz/");
 	mMsgPath = homeDir + wxT(".fahmon/");
