Index: /home/justin/workspace/pymol/modules/pymol/commanding.py
===================================================================
--- /home/justin/workspace/pymol/modules/pymol/commanding.py	(revision 3144)
+++ /home/justin/workspace/pymol/modules/pymol/commanding.py	(working copy)
@@ -198,11 +198,11 @@
                 _self.unlock(0,_self)
             r = DEFAULT_SUCCESS
             if show_splash==1: # generic / open-source
-                png_path = _self.exp_path("$PYMOL_PATH/data/pymol/splash.png")
+                png_path = _self.exp_path("$PYMOL_DATA/data/pymol/splash.png")
             elif show_splash==2: # evaluation builds
-                png_path = _self.exp_path("$PYMOL_PATH/data/pymol/epymol.png")
+                png_path = _self.exp_path("$PYMOL_DATA/data/pymol/epymol.png")
             else: # incentive builds
-                png_path = _self.exp_path("$PYMOL_PATH/data/pymol/ipymol.png")
+                png_path = _self.exp_path("$PYMOL_DATA/data/pymol/ipymol.png")
             if os.path.exists(png_path):
                 _self.do("_ cmd.load_png('%s',0,quiet=1)"%png_path)
         else: