--- Makefile	2009-01-12 07:44:39.000000000 +0100
+++ Makefile.new	2009-06-17 22:58:39.000000000 +0200
@@ -51,7 +51,7 @@
 ### MAIN PROGRAM
 
 $(OUTFILE): axel.o conf.o conn.o ftp.o http.o search.o tcp.o text.o
-	$(CC) *.o -o $(OUTFILE) $(LFLAGS)
+	$(CC) $(LDFLAGS) *.o -o $(OUTFILE) $(LIBS)
 ifndef DEBUG
 	-$(STRIP) $(OUTFILE)
 endif
--- Makefile	2009-06-17 23:03:37.000000000 +0200
+++ Makefile.new	2009-06-17 23:05:03.000000000 +0200
@@ -57,7 +57,7 @@
 endif
 
 .c.o:
-	$(CC) -c $*.c -o $*.o -Wall $(CFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $*.c -o $*.o
 
 install-bin:
 	mkdir -p $(DESTDIR)$(BINDIR)/
