diff -Nu -Nu -uarN ccp4-6.0.2.orig/src/Makefile.in ccp4-6.0.2/src/Makefile.in
--- ccp4-6.0.2.orig/src/Makefile.in	2008-04-13 10:42:55.000000000 +0200
+++ ccp4-6.0.2/src/Makefile.in	2008-04-13 10:44:14.000000000 +0200
@@ -45,7 +45,7 @@
 OTHERS = rsps anglen secstr proclean pplot gfac2pdb tplot mplot probplot rmsdev nb \
          scala dm dmmulti stereo mapmask maprot molrep ncsmask rdent restrain \
          sfcheck sftools tlsanl tlsextract topp toplist dyndom sc combat mlphare fffear ffjoin \
-         anisoanl refmac5 libcheck beast dtrek2scala
+         anisoanl libcheck beast dtrek2scala
 
 # targets built from .c sources:
 CSIMPLE = pltdev binsort
diff -Nu -Nu -uarN ccp4-6.0.2.orig/x-windows/Makefile.in ccp4-6.0.2/x-windows/Makefile.in
--- ccp4-6.0.2.orig/x-windows/Makefile.in	2008-04-13 10:42:32.000000000 +0200
+++ ccp4-6.0.2/x-windows/Makefile.in	2008-04-13 10:49:21.000000000 +0200
@@ -35,7 +35,7 @@
 #
 # all
 #
-all : xdlview libjwc_c libjwc_f rotgen_ hklview ipdisp.exe xdlgjk xjiffy rasmol2 ipmosflm
+all : xdlview libjwc_c libjwc_f rotgen_ hklview ipdisp.exe xdlgjk xjiffy
 #
 # xdl_view
 #
@@ -156,7 +156,7 @@
 # install - not yet installed!
 #
 install_obj = ipdisp.exe hklview xdldataman xdlmapman \
-rotgen ipmosflm
+rotgen
 install: all
 	$(INSTALL_PROGRAM) $(srcdir)/ipdisp.exe $(bindir)
 	$(INSTALL_PROGRAM) $(srcdir)/ipdisp/ipdisp $(bindir)
@@ -164,9 +164,7 @@
 	$(INSTALL_PROGRAM) $(srcdir)/xdldataman $(bindir)
 	$(INSTALL_PROGRAM) $(srcdir)/xdlmapman $(bindir)
 	cd $(srcdir)/XCCPJIFFY ; $(MAKE) install
-	$(INSTALL_PROGRAM) $(rasmol_dir)/rasmol $(bindir)
 	cd $(rotgen_dir); $(MAKE) install
-	$(INSTALL_PROGRAM) $(mosflm_dir)/bin/ipmosflm $(bindir)
 #
 # clean
 #
@@ -180,14 +178,12 @@
 	cd $(rotgen_dir) ; $(MAKE) clean ;\
 	cd $(libjwc_c_dir) ; $(MAKE) clean ;\
 	cd $(libjwc_f_dir) ; $(MAKE) clean ;\
-	cd $(mosflm_dir) ; $(MAKE) realclean
 distclean: realclean
 	-rm -rf Makefile config.status ;\
 	cd $(xdlview_dir) ; $(MAKE) distclean ;\
 	cd $(rotgen_dir) ; $(MAKE) distclean ;\
 	cd $(libjwc_c_dir) ; $(MAKE) distclean ;\
 	cd $(libjwc_f_dir) ; $(MAKE) distclean ;\
-	cd $(mosflm_dir) ; $(MAKE) distclean
 uninstall: 
 	for i in $(install_obj); do \
 	  rm -f $(bindir)/$$i ;\
@@ -196,7 +192,6 @@
 	cd $(xdlview_dir) ; $(MAKE) uninstall ;\
 	cd $(libjwc_c_dir) ; $(MAKE) uninstall ;\
 	cd $(libjwc_f_dir) ; $(MAKE) uninstall
-	cd $(mosflm_dir) ; $(MAKE) uninstall
 	test -f $(libdir)/font.bin && rm -f $(libdir)/font.bin ; \
 	test -f $(libdir)/nice_font.bin && rm -f $(libdir)/nice_font.bin
 
