--- ccp4-5.99.5.orig/x-windows/Makefile.in	2005-12-05 03:06:57.000000000 -0800
+++ ccp4-5.99.5/x-windows/Makefile.in	2005-12-05 03:07:23.000000000 -0800
@@ -35,7 +35,7 @@
 #
 # all
 #
-all : xdlview libjwc_c libjwc_f rotgen_ hklview ipdisp.exe xdlgjk xjiffy rasmol2 ipmosflm
+all : xdlview libjwc_c libjwc_f rotgen_ hklview ipdisp.exe xdlgjk xjiffy
 #
 # xdl_view
 #
@@ -161,7 +161,6 @@
 	$(INSTALL_PROGRAM) $(srcdir)/xdldataman $(bindir)
 	$(INSTALL_PROGRAM) $(srcdir)/xdlmapman $(bindir)
 	cd $(srcdir)/XCCPJIFFY ; $(MAKE) install
-	$(INSTALL_PROGRAM) $(rasmol_dir)/rasmol $(bindir)
 	cd $(rotgen_dir); $(MAKE) install
 	$(INSTALL_PROGRAM) $(mosflm_dir)/bin/ipmosflm $(bindir)
 	cd $(xdlview_dir); $(MAKE) install
--- ccp4-6.0.2.orig/src/Makefile.in     2008-03-02 17:59:15.000000000 +0100
+++ ccp4-6.0.2/src/Makefile.in  2008-03-02 18:01:35.000000000 +0100
@@ -45,7 +45,7 @@
 OTHERS = rsps anglen secstr proclean pplot gfac2pdb tplot mplot probplot rmsdev nb \
          scala dm dmmulti stereo mapmask maprot molrep ncsmask rdent restrain \
          sfcheck sftools tlsanl tlsextract topp toplist dyndom sc combat mlphare fffear ffjoin \
-         anisoanl refmac5 libcheck beast dtrek2scala
+         anisoanl beast dtrek2scala
 
 # targets built from .c sources:
 CSIMPLE = pltdev binsort